docs/frequently-asked-questions.md

```diff
@@ -16,68 +16,68 @@
 
 ## Configuration
 
-### I get an error-message saying "Something went wrong, try reload the page", what should I do?
+### If I get an error-message saying "Something went wrong, try reloading the page", what should I do?
 
 First, let's identify your stack.
 
-* Are you using Ushahidi.io and running a deployment there? If YES, please contact our team through one of these channels [https://www.ushahidi.com/contact\#](https://www.ushahidi.com/contact#)
-* Are you a developer, setting up Ushahidi yourself, maybe for the first time? Then start by checking the network tab in your development browser of choice and identifying any network errors. Look at the response for each and check waht you see. 
+* Are you using ushahidi.io and running a deployment there? If YES, please contact our team through one of these channels: [https://www.ushahidi.com/contact\#](https://www.ushahidi.com/contact#)
+* Are you a developer, setting up Ushahidi yourself? Start by checking the network tab in your development browser of choice and identifying any network errors. Look at the response for each and check what you see. 
 
-  The most common reasons for this error are:
+  The most common reasons for this error:
 
-  * you have used the wrong url in the BACKEND\_URL key of your platform-client .ENV file. Check that when you access the URL in the browser + /api/v3/config it returns valid json. It should look like this when you call the /api/v3/config endpoint for your API: [https://qa.api.ushahidi.io/api/v3/config](https://qa.api.ushahidi.io/api/v3/config) 
-  *  the server is failing for some reason. If the server is failing, it will likely show an error either in the browser when you call the API URL or in the server logs, which you can see in files contained within {the\_platform\_install\_dir}/storage/logs. Check the errors in the log, often you will see that there is a permissions error somewhere, or a directory is missing, and will be able to solve it yourself.  
+  * You have used the wrong url in the BACKEND\_URL key of your platform-client .ENV file. Check that when you access the URL in the browser + /api/v3/config it returns a valid json. It should look like this when you call the /api/v3/config endpoint for your API: [https://qa.api.ushahidi.io/api/v3/config](https://qa.api.ushahidi.io/api/v3/config) 
+  *  The server is failing for some reason. If the server is failing, it will likely show an error either in the browser when you call the API URL or in the server logs, which you can see in files contained within {the\_platform\_install\_dir}/storage/logs. Check the errors in the log-- often you will see that there is a permissions error somewhere, or a directory is missing, which you can solve yourself.  
```

Comment:
not sure why the comma was changed to `--` here, can you explain? (I may be missing something, sorry!)